Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -738,8 +742,12 @@ bool RenderLayerCompositor::updateCompositingLayers(CompositingUpdateType update
         needHierarchyUpdate |= layersChanged;
     }
 
-#if !LOG_DISABLED
```

Comment:
why can't we remove compositingLogEnabled() and keep the logic simple? Do we really need this runtime check?

Reply:
We cannot remove the compositingLogEnabled() simply.
Because this functionality takes 0.29ms to 6.39ms time at the runtime (worst case 6.39ms).
so, whoever wants it, he enables and uses it dynamically at runtime.

```
2018 Jan 22 11:59:28.677461 arrisxg1v3 runXRE[183]: Total layers   primary   secondary   obligatory backing (KB)   secondary backing(KB)   total backing (KB)  update time (ms)
2018 Jan 22 11:59:28.679009 arrisxg1v3 runXRE[183]: 1           1         0              3600.00                   0.00                3600.00               0.35
```
```
2018 Jan 22 12:00:20.951033 arrisxg1v3 runXRE[183]: Total layers   primary   secondary   obligatory backing (KB)   secondary backing(KB)   total backing (KB)  update time (ms)
2018 Jan 22 12:00:20.952795 arrisxg1v3 runXRE[183]: 12           3         9              6970.00                 312.39                7282.39               5.74
```